.github/scripts/trymerge.py

```diff
@@ -1238,6 +1238,10 @@ def handle_exception(e: Exception, msg: str = "Merge failed") -> None:
         run_url = os.getenv("GH_RUN_URL")
         if run_url is not None:
             msg += f"\nRaised by {run_url}"
+        if land_checks:
```

Comment:
I think we should be more descriptive with what merge -g does (and how it's different) and same with merge -f. Maybe we can update our wiki to describe these in more detail and then just link to that wiki here as well.